### PR TITLE
Initial rebrand

### PR DESCRIPTION
**Pull request by JulianGro**
This PR does an initial rebrand from GitAhead to Gittyup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
-Report bugs in GitAhead by opening an issue in the
-[issue tracker](https://github.com/gitahead/gitahead/issues).
+Report bugs in Gittyup by opening an issue in the
+[issue tracker](https://github.com/Murmele/gitahead/issues).
 Remember to search for existing issues before creating a new one.
 
-If you still need help, email us at
-[support@gitahead.com](mailto:support@gitahead.com).
+If you still need help, check out our Matrix channel
+[GitAheadFork:martix.org](https://matrix.to/#/#GitAheadFork:matrix.org).
```
Comment (Murmele) [inline README.md:27]: I replaced the Matrix channel by the new name: #Gittyup:matrix.org

Comment (Murmele) [reply README.md:27]: Sorry I don't know yet how to change it my self in the review view. I will check it out

File: pack/flatpak/io.github.gittyup.gittyup.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+				"install -Dm755 ../rsrc/Gittyup.iconset/icon_128x128@2x.png /app/share/icons/hicolor/128x128@2x/apps/gittyup.png",
+				"install -Dm755 ../rsrc/Gittyup.iconset/icon_256x256.png /app/share/icons/hicolor/256x256/apps/gittyup.png",
+				"install -Dm755 ../rsrc/Gittyup.iconset/icon_256x256@2x.png /app/share/icons/hicolor/256x256@2x/apps/gittyup.png",
+				"install -Dm755 ../rsrc/Gittyup.iconset/icon_512x512.png /app/share/icons/hicolor/512x512/apps/gittyup.png"
+			],
+			"sources": [
+				{
+					"type": "git",
+					"branch": "master",
+					"url": "https://github.com/Murmele/gitahead"
```
Comment (Murmele) [inline pack/flatpak/io.github.gittyup.gittyup.json:39]: I will try to create a new repo, so we can change the name here too. I will have a look in that. Nothing to do for you

**Review by Murmele — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JulianGro) [reply README.md]: 1Auswahl_944_e
This is how you ideally suggest changes to a PR. I can then just accept the change or make an additonal change to the change.

**Pull request merged**